### ISSUE TITLE
Android Status Bar wrong colour

### ISSUE THREAD

**Issue by tomy-pg** [pg-irc/pathways-frontend#1142]
![Screenshot_20200429-152844_Arrival Advisor.jpg](https://images.zenhubusercontent.com/5c6c8cff9d62b128eab447ff/612d725c-d35f-469a-b9be-cb8493560391)

The status bar for Android is being set to a solid blue/purple background colour. This is a **known issue** that I have subscribed to on Expo's Github page [here](https://github.com/expo/expo/issues/7635). The contributors of this discussion have experienced what we see in our app and what I have seen while tackling this issue. 

**What I tried** (also the recommended configuration suggested by Expo [here](https://docs.expo.io/guides/configuring-statusbar/?redirected) ):
In `app.json`:
```
"androidStatusBar": {
    "barStyle": "light-content", // default value
    "backgroundColor": "#00000088" // semi-transparent black for `light-content` 
  },
```
and in `src/components/main/main_component.tsx`
```
export const MainComponent = (props: Props): JSX.Element => {
...
StatusBar.setBarStyle('light-content');
StatusBar.setBackgroundColor('#00000088');
```
**What I saw**:
1. When loading the JS Bundle, the StatusBar aligned correctly with what was set in `app.json`.
2. On initial mount of the homepage, the StatusBar still shows as being solid blue/purple.
3. The StatusBar changes background colour to be consistent with the configuration only once I do an action on the i.e switch screens, bookmark a topic.

### Acceptance Criteria 
The following StatusBar properties are consistent throughout usage:
- [ ]  translucent: true
- [ ] barStyle: 'light-content'
- [ ] backgroundColor: '#00000088'

### Steps for QA: 
1.  Load the app:
- [ ] ensure the status bar has the properties found in Acceptance Criteria
2. Once the app loads:
- [ ] ensure the status bar has the properties found in Acceptance Criteria
3. Upon navigating to different screens:
- [ ] ensure the status bar has the properties found in Acceptance Criteria

**Comment by rasmus-storjohann-PG** [comment]
Waiting for Expo to fix an issue on their end, then we can proceed to fix it on our

**Comment by tomy-pg** [comment]
I believe I can fix this once Expo SDK 38 is merged into our codebase. They have released a new `expo-status-bar` that we can use that works well with the `expo-cli`.

**Comment by ian-lin-PG** [comment]
From what I observe the blue header is no longer present, adding screenshots of three different screens that have status bar that blends in with the header colour.
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-07-20 at 13 03 55](https://user-images.githubusercontent.com/52009863/87981278-d4ef3c00-ca89-11ea-81a7-2110eb14cf9a.png)
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-07-20 at 13 03 49](https://user-images.githubusercontent.com/52009863/87981281-d6b8ff80-ca89-11ea-8430-b0440b260540.png)
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-07-20 at 13 03 45](https://user-images.githubusercontent.com/52009863/87981284-d7ea2c80-ca89-11ea-8b7f-b33f441f6fbb.png)

**Comment by rasmus-storjohann-PG** [comment]
I still see the purple header in Expo. This seems like a low risk change, so I'd be fine to merge it and see if it's fixed in a real build, in case you think that might be the issue.

**Comment by tomy-pg** [comment]
Native-bases optional `androidStatusBarColor` property for their `Header` component was overwriting our StatusBar colour.

**Comment by ian-lin-PG** [comment]
Here are the screenshots from the latest PR which shows a darker gradient on the main color.
![Screenshot_20200804-094227](https://user-images.githubusercontent.com/52009863/89321394-c2533600-d637-11ea-89db-0360e679d674.jpg)
![Screenshot_20200804-094219](https://user-images.githubusercontent.com/52009863/89321402-c54e2680-d637-11ea-95a8-7330e353b937.jpg)

**Comment by ian-lin-PG** [comment]
QA passed 1.43.0, status bar is teal by default.
![Screenshot_20200817-101809](https://user-images.githubusercontent.com/52009863/90424798-8ffffa80-e073-11ea-84f7-af129ff1cfcb.jpg)
